projects/tech-zone/new-development-url-migration-guide.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+---
+# New Tech Zone GitHub URL - Migration Guide
+
+**Author:** [Phil Wiffen](https://twitter.com/phil_wiffen)
+
+## Overview
+
+This guide covers how to migrate your Tech Zone development from using the martinzugec-ctx organisation, to the citrixtechzone Github location, if you're using using GitHub Desktop and Visual Studio Code.
+
+## Conceptual steps
+
+You will need to change your git development environment so that it points to the new github location.
+
+To do this you’ll need to:
+
+1.  Clone the new repository in GitHub Desktop, which will get the new location on your local computer
+1.  Remove the old repository from your local file system – so that you don’t accidentally develop in there once migrated.
+
+## Clone the new repository in Github desktop
+
+1.  Open GitHub Desktop
+1.  Either press Ctrl+T to bring up your repositories, or click on the "Current repository" drop down in the UI
```

Comment:
Either use mouse (image with highlighted control to select), or include macOS keyboard shortcuts.

UPDATE: Just finished reading the second half of the sentence :) I would just switch the order (mouse first as it's applicable to everyone).

Reply:
I made it mouse-only :)